### PR TITLE
Fixed small bug of ASATs  count

### PR DESCRIPTION
**Pull request by boninggong**

### CODE REVIEW

File: visualization/codemirror/sourceCode.js [hunk 1]
```diff
@@ -97,7 +97,9 @@ var sourceCode = (function() {
 	 * Clear all three arrays
 	 */
 	function emptyArrays() {
-		currentAsatWarnings, currentCatWarnings, currentMessageWarnings = [];
+		currentAsatWarnings = [];
```
Comment (ClintonCao) [inline visualization/codemirror/sourceCode.js]: Why was this change needed? Is it because you they are referencing to the same array?

Comment (boninggong) [inline visualization/codemirror/sourceCode.js]: The method using the comma caused a bug, that's why I changed back to the old way.

Comment (ClintonCao) [inline visualization/codemirror/sourceCode.js]: Hmm okay then. Oh JavaScript...

**Comment by ClintonCao** [comment]
I will merge this then!
